### ISSUE TITLE
create-app build error - graphql-language-service-utils

### ISSUE THREAD

**Issue by regicsolutions** [backstage/backstage#3961]
Unable to compile with latest Scaffolded Backstage App template, also getting this same error on an existing app i just upgraded to the latest release 

```
$ yarn build
yarn run v1.22.1
$ lerna run build
lerna notice cli v3.22.1
lerna info Executing command in 2 packages: "yarn run build"
lerna ERR! yarn run build exited 1 in 'app'
lerna ERR! yarn run build stdout:
$ backstage-cli app:build
Loaded config from app-config.yaml, app-config.local.yaml
Failed to compile.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:

Error: Failed to compile.
C:/User/node_modules/graphql-language-service-utils/esm/fragmentDependencies.js
Module not found: Can't resolve 'nullthrows' in 'C:\User\node_modules\graphql-language-service-utils\esm'

error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in 'app'
error Command failed with exit code 1.
```

build should be successful 

## Steps to Reproduce

`yarn build`

## Context

- Node 12
- Win 10
- Chrome/Edge

**Comment by adamdmharvey** [comment]
Also caught in `master` with the end-to-end test as well:
https://github.com/backstage/backstage/pull/3958/checks?check_run_id=1663428006

**Comment by andrewthauer** [comment]
Same issue for me on mac with node 14

**Comment by Rugvip** [comment]
Fixed by https://github.com/graphql/graphiql/pull/1753, released in `graphql-language-service-utils` `v2.5.1`

If you're in a new app and see this issue, you'll want to bump `graphql-language-service-utils` in the lockfile. Builds in this repo should be fine now.
